tests/e2e/v2store_deprecation_test.go

```diff
@@ -125,15 +129,12 @@ func TestV2DeprecationSnapshotMatches(t *testing.T) {
 	assert.NoError(t, epc.Close())
 
 	assertSnapshotsMatch(t, oldMemberDataDir, newMemberDataDir, func(data []byte) []byte {
-		// Patch cluster version
-		data = bytes.Replace(data, []byte("3.5.0"), []byte("X.X.X"), -1)
-		data = bytes.Replace(data, []byte("3.6.0"), []byte("X.X.X"), -1)
 		// Patch members ids
 		for i, mid := range members1 {
-			data = bytes.Replace(data, []byte(fmt.Sprintf("%x", mid)), []byte(fmt.Sprintf("member%d", i+1)), -1)
```

Comment:
Why change this?

Reply:
please refer to the comment above - https://github.com/etcd-io/etcd/pull/16418#issuecomment-1679344123.